### PR TITLE
correct issue of 3.3.1

### PR DESCRIPTION
**Pull request by slubwama**

### CODE REVIEW

File: omod/src/main/webapp/portlets/login.jsp [hunk 1]
```diff
@@ -41,8 +41,7 @@
 				<input type="hidden" name="redirect" value="" />
 			</c:otherwise>
 		</c:choose>
-		
-		<input type="hidden" name="refererURL" value='<request:header name="referer" />' />
+		<input type="hidden" name="refererURL" value='<c:out value="${pageContext.request.getHeader('referer')}"/>' />
```
Comment (dkayiwa) [inline omod/src/main/webapp/portlets/login.jsp:44]: Does this mean that the XSS attack came via the referer param?

Comment (slubwama) [reply omod/src/main/webapp/portlets/login.jsp:44]: Yes

**Comment by ibacher** [comment]
Could we also do the same for this one: https://github.com/openmrs/openmrs-module-legacyui/blob/528e938e454f6c11ec6b620ce651cc6fb634ec2a/omod/src/main/webapp/admin/patients/mergePatientsForm.jsp#L784

**Comment by slubwama** [comment]
> Could we also do the same for this one:
> 
> https://github.com/openmrs/openmrs-module-legacyui/blob/528e938e454f6c11ec6b620ce651cc6fb634ec2a/omod/src/main/webapp/admin/patients/mergePatientsForm.jsp#L784

Done

**Review by ibacher — APPROVED** [review]

**Pull request merged**